src/modules/main/types/experts.ts

```diff
@@ -0,0 +1,39 @@
+export interface IExperts {
+    userId: number,
+    emailId: string,
+    password: string,
+    status: ExpertStatus,
+    firstName: string,
+    secondName: string,
+    qualification: string,
+    phone: string,
+    bio: string,
+    createdAt: Date,
+}
+
+enum ExpertStatus {
+    'new',
```

Comment:
enum should be like this:
https://github.com/ita-social-projects/dokazovi-fe/blob/f3a08593d28e4a59ef9f24f32d2abc11bf2039eb/src/modules/direction/store/actionTypes.ts#L1